docs/getting_started/index.md

```diff
@@ -6,7 +6,7 @@ Zeus automatically tunes the **batch size** and **GPU power limit** of a recurri
     Zeus can optimize the batch size of **recurring** jobs, i.e. training jobs that re-run multiple times over time. However, Zeus can still optimize the GPU power limit even if your jobs does not recur.
 
 !!! Info
-    Zeus currently only supports **single GPU training**. Support for single-node data parallel training and distributed data parallel training will be added soon.
+    Zeus currently supports **single GPU training** and **single node data parallel training**. Support for distributed data parallel training will be added soon.
```

Comment:
I tested with `examples/imagenet/train.py` by setting the `--gpu` argument. In this case, it will just run training with one GPU. It did work. But I have not tested the other two examples. Will do it!